R/style-roxygen-examples.R

```diff
@@ -3,28 +3,20 @@
 #' Finds the start and stop indices of the lines in `text` that are
 #' code examples in roxygen comments.
 #' @param text
-identify_start_and_stop_of_royxgen_examples_from_text <- function(text) {
-  starts <- identify_start_points(text)
-  stop_candidates <- identify_stop_candidates(text)
-  stops <- map_int(starts, match_stop_to_start,
-    stop_candidates = stop_candidates
-  )
+identify_start_stop_of_roxygen_examples_from_text <- function(text) {
+  starts <- grep("^#'\\s@examples", text, perl = TRUE)
```

Comment:
why not capsule it into separate function? Also, probably the pattern must be modified slightly.

Reply:
Ok, as far as capsuling goes, let's stick to what you proposed.